deploy/hooks/post-restore-code

```diff
@@ -17,8 +23,8 @@ if [ -f rc_$TARGET ]; then
         if [ -f rc_branch ]; then
             source rc_branch
         fi
+        make all
```

Comment:
as discussed you need to build with the correct variables and if we disable the tests it won't be the case for sure.